src/Compilers/Core/Portable/Syntax/SyntaxTree.cs

```diff
@@ -284,14 +284,27 @@ internal virtual FileLinePositionSpan GetMappedLineSpanAndVisibility(TextSpan sp
         internal string GetDisplayPath(TextSpan span, SourceReferenceResolver? resolver)
         {
             var mappedSpan = GetMappedLineSpan(span);
-            if (resolver == null || mappedSpan.Path.IsEmpty())
+            if (resolver is null || mappedSpan.Path.IsEmpty())
             {
                 return mappedSpan.Path;
             }
 
             return resolver.NormalizePath(mappedSpan.Path, baseFilePath: mappedSpan.HasMappedPath ? FilePath : null) ?? mappedSpan.Path;
         }
 
+        /// <summary>
+        /// Returns the path used for emit purposes.
+        /// </summary>
+        internal string GetDisplayPath(SourceReferenceResolver? resolver)
```

Comment:
The doc comment says this is for emit, but the method name says it's for display. Should the method name be changed to something like `GetMappedPath`?

Reply:
The term _mapped_ already has a bit of a meaning in our APIs. Essentially takes into account `#line` directives. Concerned about overloading that a bit. Going to use `GetNormalizedPath` as _normalize_ is already the term for taking into account `/pathmap` arguments